ui-v2/app/services/data-source/protocols/http.js

```diff
@@ -3,10 +3,11 @@ import { get } from '@ember/object';
 
 export default Service.extend({
   datacenters: service('repository/dc'),
+  namespaces: service('repository/nspace'),
   token: service('repository/token'),
   type: service('data-source/protocols/http/blocking'),
   source: function(src, configuration) {
-    const [, dc /*nspace*/, , model, ...rest] = src.split('/');
+    const [, , /*nspace*/ dc, model, ...rest] = src.split('/');
```

Comment:
Did you leave an extra comma here?

Reply:
Oh no hehehe, I just tried to fix this about 4 times before I realised prettier is 'unfixing' it for me whenever I commit! 🤣 

I've got a PR coming soon that uncomments this `nspace` so the problem will go away when I PR that.